ux.symfony.com/templates/_footer.html.twig

```diff
@@ -0,0 +1,48 @@
+<footer>
+    <div class="container-fluid container-xxl py-5 px-lg-5">
+        <div class="row align-items-center" style="--bs-gutter-x: 0;">
+            <div class="col-12 col-xl-3 order-xl-3 text-center text-xl-end">
+                <a href="https://github.com/symfony/ux" rel="external noopener noreferrer"
+                   style="font-size: 1.5rem;">
```

Comment:
The code style I prefer is, when you have multiline attributes, to put the `>` on its own line:

```
<a href="https://github.com/symfony/ux" rel="external noopener noreferrer"
    style="font-size: 1.5rem;"
>
    <twig:Icon name="github"/>
```

My reasoning is that the `>` on its own line makes the next line (`<twig:Icon>`) look indented. WDYT?

Reply:
100% agree